test/int/project_test.go

```diff
@@ -124,6 +126,25 @@ var _ = Describe("AtlasProject", func() {
 		})
 	})
 
+	Describe("Deleting the project (not cleaning Atlas)", func() {
+		It("Should Succeed", func() {
+			By(`Creating the project with retention policy "keep" first`, func() {
+				createdProject = mdbv1.DefaultProject(namespace.Name, connectionSecret.Name)
+				createdProject.ObjectMeta.Annotations = map[string]string{customresource.ResourcePolicyAnnotation: customresource.ResourcePolicyKeep}
+				Expect(k8sClient.Create(context.Background(), createdProject)).ToNot(HaveOccurred())
+
+				Eventually(testutil.WaitFor(k8sClient, createdProject, status.TrueCondition(status.ReadyType)),
+					ProjectCreationTimeout, interval).Should(BeTrue())
+			})
+			By("Deleting the project", func() {
+				Expect(k8sClient.Delete(context.Background(), createdProject)).To(Succeed())
+				time.Sleep(10 * time.Second)
+				Expect(checkAtlasProjectRemoved(createdProject.Status.ID)()).Should(BeFalse())
+				createdProject = nil
```

Comment:
[q] shouldn't we remove this project from Atlas to keep it cleaner?

Reply:
I thought the project cleaner will do it but better do manually, right 👍